Week2/Solution/solution.txt

```diff
@@ -0,0 +1,46 @@
+1. Create table 
+A)Student table 
+
+CREATE TABLE student
+(
+id int PRIMARY KEY,
+name text
+);
+
+B)Student detail table
+
+CREATE TABLE stud_detail
+(
+sid,
+city text,
+mob_no number,
+FOREIGN KEY (sid)  

```

Comment:
foreign key is not working

Reply:
yes, foreign key was not enabled... some changes are made now to avoid that